wemake_python_styleguide/cli/commands/explain/message_formatter.py

```diff
@@ -0,0 +1,68 @@
+"""Provides tools for formatting explanations."""
+
+from typing import Final
+
+from wemake_python_styleguide.cli.commands.explain.violation_loader import (
+    ViolationInfo,
+)
+
+_DOCS_URL: Final = 'https://pyflak.es/{0}'
```

Comment:
Can we reuse it? `formatter.py` also has it: https://github.com/wemake-services/wemake-python-styleguide/blob/master/wemake_python_styleguide/formatter.py#L47

Let's move it to `constants.py`